recipes/lfzip-nlms/meta.yaml

```diff
@@ -0,0 +1,42 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+{% set name = "LFZip" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/shubhamchandak94/LFZip/archive/v{{ version }}.tar.gz
+  sha256: ef9b9b00f984daab9d65927232a5c0b2c7f21059fd7b26070aed02215f167501
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - make
+  run:
+    - python
```

Comment:
```suggestion
    - python >=3
```

Reply:
I got a linting error and am not sure how to correct it. Should I add noarch?
Non noarch packages should have python requirement without any version constraints.

Reply:
You can ignore the linter error. It's buggy.

Reply:
Thanks!